src/pages/workspaces/workspace/workflows/FailureRerunner.js

```diff
@@ -12,17 +11,13 @@ import { rerunFailuresStatus } from 'src/libs/state'
 import * as Utils from 'src/libs/utils'
 
 
-const ToastMessageComponent = Utils.connectStore(rerunFailuresStatus, 'status')(class ToastMessageComponent extends Component {
-  render() {
-    const { status: { done, text } } = this.props
-
-    return div({ style: { padding: '0.5rem 0', display: 'flex', alignItems: 'center', fontSize: 14 } }, [
-      done ?
-        icon('success-standard', { size: 24, style: { color: colors.success(), marginRight: '1rem' } }) :
-        spinner({ style: { marginRight: '1rem' } }),
-      text
-    ])
-  }
+const ToastMessageComponent = Utils.connectStore(rerunFailuresStatus, 'status')(({ status: { done, text } }) => {
```

Comment:
Now that this is a function component, we can use the `useStore` hook instead of the wrapper.